azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/sqlserver/AEConstants.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.autoconfigure.sqlserver;
+
+public class AEConstants {
+    public static final String PROPERTY_AE_ENABLED = "spring.datasource.alwaysencrypted";
+    public static final String PROPERTY_DATASOURCE_COL_ENCRYPT =
```

Comment:
I see `DataSourcePropertyies` has prefix `spring.datasource`. But I'm confusing why here still have extra `dataSourceProperties`.

Reply:
not sure what is the question? "spring.datasource.dataSourceProperties" is needed to pass settings to hikari..